src/LibraryManager.Vsix/Json/TextviewCreationListener.cs

```diff
@@ -107,22 +107,27 @@ public void VsTextViewCreated(IVsTextView textViewAdapter)
 
         private void OnFileSaved(object sender, TextDocumentFileActionEventArgs e)
         {
-            if (e.FileActionType == FileActionTypes.ContentSavedToDisk)
+            var textDocument = (ITextDocument) sender;
```

Comment:
if sender isn't `ITextDocument` this will throw an InvalidCastException.  Did you want `sender as ITextDocument`?